### DISCUSSION TITLE
Suggest: Make "export" on Excel facade public instead of protected

### DISCUSSION THREAD

**Discussion by GregPeden** [spatie/laravel-medialibrary#1932] [Ideas] [answered]
Is there a good reason to guard the "export" method on the Excel facade?

I have a case where it would be useful. I would like to retrieve the document as a temporary file to then handle it by a means not available by the other facade methods. My alternative at present is to save it to the local drive temporarily, process it, then delete the file. A temporary file would be simple and more tolerant of runtime faults.

Any thoughts?

**Comment by freekmurze** [comment, marked as answer]
Does this package contain an excel facade?

**Reply by GregPeden** [reply]
Yep, you're right, I posted this in the wrong Git repo. ;)

For reference, I meant to put it here:

https://github.com/Maatwebsite/Laravel-Excel

And this package I am using for management of remote storage.
